api/open_saves.proto

```diff
@@ -360,6 +360,9 @@ message QueryRecordsRequest {
   // List of sort orders to return records. These SortOrders are applied 
   // in sequence.
   repeated SortOrder sort_orders = 5;
+
+  // the limit of the number of records to return.
+  int64 limit = 6;
```

Comment:
I think int32 makes more sense here because you're casting limit to int later, which could be 32 bits on some platforms. Also I don't think we'd ever use a limit bigger than 20 billion.